tests/integration/tests/user_scenarios/duplicate_different_module_version.rb

```diff
@@ -6,12 +6,17 @@
 # Init
 local_files_root_path = ENV['MANIFESTS'] || 'tests/manifests'
 
-dsc_source_module_path = 'C:\ProgramData\PuppetLabs\puppet\cache\lib\puppet_x\dsc_resources\xPSDesiredStateConfiguration'
+puppet_agent_cache_path = 'C:\ProgramData\PuppetLabs\puppet\cache\lib\puppet_x'
```

Comment:
We should be asking Puppet for these values instead of hardcoding them - i.e.  `agent.puppet['vardir']`